packages/core/src/2d/sprite/Sprite.ts

```diff
@@ -0,0 +1,260 @@
+import { Vector2, Vector4 } from "@oasis-engine/math";
+import { RefObject } from "../../asset/RefObject";
+import { Engine } from "../../Engine";
+import { Texture2D } from "../../texture";
+
+// A 2D Rectangle defined by X and Y position, width and height.
```

Comment:
Write comments according to specifications

Reply:
> Write comments according to specifications

done，has removed rect from sprite to math